README.md

```diff
@@ -37,6 +42,98 @@ php bin/magento cache:clean
+
+![Image selection](https://assets.imgix.net/sdk/magento/08-image_selection.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
+
+Users may select one or more images from this view by clicking on it. Users will notice a light blue highlight around images that have been selected. To deselect an image, click on it once more until the highlight disappears. Once done, press the Add Image button to insert the selected image(s) to the product. 
+
+![Almost finished adding an image](https://assets.imgix.net/sdk/magento/09-image_added.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
+
+At this point, users may select Save to finalize these changes.
+
+# Add Images to Pages
+
+In addition to Product images, the imgix extension can also be used to add images to Pages and/or Blocks. Navigate to either the Page or Block editors via the Content tab on the admin sidebar, creating a new page/block or editing an existing one as desired.
+
+**Note:** Because this extension is created in part for us on Page Builder, users should ensure that they are using this extension for Magento version 2.4.3 and later.
+
+![The page view in MAgento](https://assets.imgix.net/sdk/magento/09-image_added.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
+
+To edit an existing page, press the **Select** action and then the **Edit** option. Under the **Content** section, select the **Edit with Page Builder** button. 
+
+![Editing with Page Builder](https://assets.imgix.net/sdk/magento/11-edit_page.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
+
+From the **Magento Page Builder**, users can create a new row/column component (located under Layout section) by dragging it onto the page in the desired location. This is where an imgix image can be added. After, users can click and drag the imgix Image component to the desired row or column.
+
+![Editing with Page Builder](https://assets.imgix.net/sdk/magento/12-editing_page_builder.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
+
+From this point, users may follow the same steps as in the Product section to browse, search, and insert an image into this component. Note: in the context of the Page Builder, only one image can be added per component.
+
+![Browsing images in Page Builder](https://assets.imgix.net/sdk/magento/13-browsing_images.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
+
+![Selecting an image in Page Builder](https://assets.imgix.net/sdk/magento/14-selecting_an_image_in_page_builder.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
```

Comment:
these images are 404ing

Reply:
whoops, forgot to upload them.

they're online now 👍